pkg/operator/controller/vulnerabilityreport.go

```diff
@@ -4,6 +4,7 @@ import (
 	. "github.com/aquasecurity/starboard/pkg/operator/predicate"
 
 	"context"
+	errors2 "errors"
```

Comment:
I think it's better to use alias for "k8s.io/apimachinery/pkg/api/errors", e.g.:

```go
import (
	"errors"

	apierrors "k8s.io/apimachinery/pkg/api/errors"
	// or
	k8sapierrors "k8s.io/apimachinery/pkg/api/errors"
)
```